doc/installer-hacks.md

```diff
@@ -105,3 +105,17 @@ equivalent, the old code is simply deleted, not commented.
 
 * Code dealing with BIOS-ID changed to blindly assume common values. Original
   code commented and marked with "storage-ng".
+
+## Changes in autoyast2-installation
+
+* Commented some parts in `AutoInstallRules` to not use old storage lib (commented
+  parts marked with `storage-ng`). This fix problem with intall process (see 
+  [this PBI](https://trello.com/c/qsxBrzIE/499-2-storageng-get-failing-openqa-test-fixed-by-adjusting-autoinstallrules-to-storage-ng)).
+
+* Several tests have been skipped ir order to create the package. Commented tests
+  need `yast2-installation`, but it is not possible to provide it due to conflict between `yast2-storage` and `yast2-storage-ng`.
+
+* `yast2-installation` requires `autoyast2-installation` that requires `yast2-update`
+  that requires `yast2-storage`.
+
+* There is a cyclic dependency between `yast2-installation` and `autoyast2-installation`.
```

Comment:
Oops, if we have a cyclic dependency, we should resolve it somehow. Please, report a bug and we can plan for it. Thanks :)